### ISSUE TITLE
​

### ISSUE THREAD

**Issue by Exponential-Workload** [twitter/the-algorithm#2023]
​

**Comment by mppbtw** [comment]
I agree, it would seem that           is a very big problem with the algorithm and we should try and fix it. perhaps we could try        or even           but im no expert in the matters of           so we would have to contact the           experts.

**Comment by MadKamel** [comment]
it took me several seconds to figure out how to open this issue

**Comment by mppbtw** [comment]
> it took me several seconds to figure out how to open this issue

this comment section shall forever be a hall of fame for those who realised you can change the url to be the issue number. the worthy. the lifekind. perchance

**Comment by Exponential-Workload** [comment]
the fact that this issue has not yet been addressed is preposterous and must be addressed immediately

**Comment by colleirose** [comment]
I have also encountered this issue

**Comment by colleirose** [comment]
> > it took me several seconds to figure out how to open this issue
> 
> this comment section shall forever be a hall of fame for those who realised you can change the url to be the issue number. the worthy. the lifekind. perchance

It'll be very funny if Twitter staff don't figure out how to do this

**Comment by realKarthikNair** [comment]
> It'll be very funny if Twitter staff don't figure out how to do this

I won't be suprised if they don't

# hail the ~lord~ zero width space!

**Comment by Exponential-Workload** [comment]
hail the zero width characters

**Comment by Exponential-Workload** [comment]
ong when is twitX gonna fix this

**Comment by realKarthikNair** [comment]
> ong when is twitX gonna fix this

@Exponential-Workload once they're able to open it

**Comment by Marioood** [comment]
Cannot reproduce on my Teletype Model 33 ASR.

**Comment by Exponential-Workload** [comment]
> Cannot reproduce on my Teletype Model 33 ASR.

Try on a 68' Xiaomi Smart-Wrist-Rest with integrated foot pedal command support
